src/test/groovy/com/jvm_bloggers/core/data_fetching/blogs/json_data/JsonBlogsDataFileValidationSpec.groovy

```diff
@@ -73,5 +75,47 @@ class JsonBlogsDataFileValidationSpec extends Specification {
         assert duplicateIds.isEmpty() : "Duplicate bookmarkable IDs found in json files: $duplicateIds"
     }
 
-}
+    def "should check that all twitter user tags starts with @ and are less than 16 characters"() {
```

Comment:
Instead of aggregating all errors in one assert you could check each entry as a separate test-case, this could be achieved by using Spock http://spockframework.org/spock/docs/1.0/data_driven_testing.html and `where` with `@Unroll` annotation and `jsonHandle << allTwitterUsersTags()` in where section.

Reply:
@tdziurko updated, please check: https://github.com/jvm-bloggers/jvm-bloggers/pull/565/commits/f82840ab120fb9dbd8691ca2a408e8a89e2bceb4

I also updated other test with bookmarkable id.
`where` clause is resolved before `setup`, so `jsonEntries` were null.
So I used `setupSpecs` instead and setup `jsonEntries` once for all test cases.
Spring context is not available in `setupSpecs`, so I do not use it now and now tests run much faster.